### PR TITLE
Update New-CsTeamsEventsPolicy.md

### PR DESCRIPTION
**Pull request by gregorionMS**
Updated EventAccessType to include correct implementations for Webinar and Town hall products respectively

### CODE REVIEW

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 1]
```diff
@@ -100,11 +100,10 @@ Accept wildcard characters: False
 ```
 
 ### -EventAccessType
-This setting governs which users can access the event registration page or the event site to register. It also governs which user type is allowed to join the session/s in the event. 
-Possible values are:
+This setting governs which users can access the event for Town hall and access the event registration page or the event site to register for Webinar. It also governs which user type is allowed to join the session/s in the event for both event types. Possible values are:
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:103]:
```suggestion
This setting governs which users can access the Town hall event and access the event registration page or the event site to register for a Webinar. It also governs which user type is allowed to join the session or sessions in the event for both event types. Possible values are:

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 2]
```diff
@@ -100,11 +100,10 @@ Accept wildcard characters: False
 ```
 
 ### -EventAccessType
-This setting governs which users can access the event registration page or the event site to register. It also governs which user type is allowed to join the session/s in the event. 
-Possible values are:
+This setting governs which users can access the event for Town hall and access the event registration page or the event site to register for Webinar. It also governs which user type is allowed to join the session/s in the event for both event types. Possible values are:
  - **Everyone**: Enables creating events to allow in-tenant, guests, federated, and anonymous (external to the tenant) users to register and join the event.
 
- - **EveryoneInCompanyExcludingGuests**: Enables creating events to allow only in-tenant users to register and join the event.
+ - **EveryoneInCompanyExcludingGuests**: For Webinar - enables creating events to allow only in-tenant users to register and join the event. For Town hall - enables creating events to allow only in-tenant users to join the event (Note: for Town hall, in-tenant users includes guests; this parameter will disable public Town halls).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:106]:
```suggestion
 - **EveryoneInCompanyExcludingGuests**: For Webinar - enables creating events to allow only in-tenant users to register and join the event. For Town hall - enables creating events to allow only in-tenant users to join the event (Note: for Town hall, in-tenant users include guests; this parameter will disable public Town halls). 

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @gregorionMS, kindly commit the minor suggested changes. Many thanks.

**Comment by get-itips** [comment]
I think this one should be closed as https://github.com/MicrosoftDocs/office-docs-powershell/pull/11358 is already merged

**Pull request closed without merging**